docs/user/communication.rst

```diff
@@ -41,99 +45,196 @@ Exemplary exercise detail page with posts:
+and reveals their highest authoritative role. Via tooltips that are shown as users hover over these icons (see images below),
+the system displays a brief explanation of that specific role. This way, the system builds trust in the author, and readers
+can tangibly confirm the origin of the shared information.
+
+Instructor Authority
+
+|instructor-user|
+
+Tutor Authority
+
+|tutor-user|
+
+Student Authority
+
+|student-user|
+
 React on Posts
---------------
+^^^^^^^^^^^^^^
+
+To foster interaction between users, we integrate the well-known emoji reaction bar.
+Each user in the course can react on any post by making use of the emoji selection button.
+The ``+`` emoji serves as the up-voting reaction, which influences the display order of posts.
 
-To foster interaction we integrate the well-known emoji reaction bar.
-Each user in the course can react on every post by making use of the emoji selection button.
-The ``+`` emoji serves as up-voting reaction, which influences the display order of posts.
+Reference Course Entities
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Users can reference to different course entities within their posts, such as other posts, course exercises, course lectures,
+and attachments of these lectures. All references are then prepended with icons which are unique to the reference's type,
+to help users distinguish them conveniently. In the image below, we see all possible types of references that can be created
+in an Artemis post.
+|post-with-references|
 
 Reference Posts
----------------
+"""""""""""""""
+
+If users want to refer to other posts, they can integrate a simple pattern including the hashtag (``#``) combined with
+the post identifier. A post's identifier is appended to the post title (such as seen in the screenshots above).
 
-If users want to refer to other posts, they can integrate a simple pattern including the hashtag (``#``) combined with the post identifier.
-A post's identifier is appended to the post title (such as seen in the screenshots above).
+When clicking a post reference used in a post's text, the user is navigated to the referenced post respectfully.
 
-When clicking a reference used in a post's text, the user is navigated to the referenced post.
+Reference Course Exercises
+""""""""""""""""""""""""""
+Users can refer to exercises of the current course, via the dropdown menu ``Exercise`` available on the posting markdown
+editor. (see image above) The following types of exercises are prepended unique icons to help users distinguish the type
+of the exercise being referenced.
+
+* Text Exercise
+* Quiz Exercise
+* Modeling Exercise
+* File Upload Exercise
+* Programming Exercise
+
+Reference Course Lectures
+"""""""""""""""""""""""""
+Users can refer to lectures of the current course, via the dropdown menu ``Lecture`` available on the posting markdown
+editor. (see image above)
+
+Reference Lecture Attachments
+"""""""""""""""""""""""""""
```

Comment:
```suggestion
Reference Lecture Attachments
"""""""""""""""""""""""""""""
```
Warning when building the docs that the underline does not match the text length

Reply:
Corrected 👍🏻